kodi.py

```diff
@@ -91,11 +91,14 @@
 def sanitize_name(media_name):
   # Normalize string
   normalized = unicodedata.normalize('NFKD', media_name).encode('ASCII', 'ignore')
-  # Strip symbols
-  m = RE_NAME_WITH_PARAM.match(normalized)
-  if m:
-    return m.group(1)
-  return media_name
+
+  # Strip things between and including brackets and parenthesis
+  removed_paren = re.sub(r'\([^)]*\)', '', normalized)
+  removed_bracket = re.sub(r'\[[^)]*\]', '', removed_paren)
```

Comment:
Should it actually strip out what's in the parentheses/brackets?  I ran into a problem with my playlists because of this before -- I had named them like:

Kids (Music Videos)
Kids (Shows)

I don't think I've seen a problem with other titles, but I can imagine it might be.

Particularly for playlists, though, it might cause failed matches merely because the user isn't likely to have enough in the slot to force it over to a "generic slot".